### PR TITLE
SEO Enhancements

### PR DESCRIPTION
**Pull request by souldanger**
- updated meta tags (description+keywords)
- added robots.txt
- added google site verification file
- added 'libwww-perl' blocker to .htaccess

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -4,4 +4,7 @@ RewriteRule ^download.html$ /downloads/ [R=301,L]
 RewriteRule ^downloads.html$ /downloads/ [R=301,L]
 RewriteRule ^download(/|$) /downloads/ [R=301,L]
 
+RewriteCond %{HTTP_USER_AGENT} libwww-perl.*
```
Comment (longsleep) [inline .htaccess]: Why is this required exactly?

File: _includes/head.html [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
         <link rel="shortcut icon" href="{{ site.baseurl }}/favicon.png" type="image/x-icon">
         <link rel="icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
+		<link rel="canonical" href="iridiumbrowser.de/" />
         
         <title>{% if page.title %}{{ site.title | escape }} | {{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
         
         <meta charset="utf-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="description" content="{% if page.description %}{{ page.description }}">{% endif %}
+		<meta http-equiv="content-language" content="DE">
```
Comment (longsleep) [inline _includes/head.html]: Wrong language?

File: robots.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# ===================================
+# Generator: http://pixelfolk.net/tools/robots
```
Comment (longsleep) [inline robots.txt]: All those comments seem overkill to me.

File: _includes/head.html (hunk 2 above)
Comment (souldanger) [inline _includes/head.html]: fixed lang="en" included in html tag as it should be...

File: .htaccess (hunk 1 above)
Comment (souldanger) [inline .htaccess]: its a SEO practice to block User-agent Libwww-perl.
Botnet scripts that automatically look for vulnerabilities in your software are sometimes identified as User-Agent libwww-perl. By blocking access from libwww-perl you can eliminate many simpler attacks.

File: robots.txt (hunk 3 above)
Comment (souldanger) [inline robots.txt]: fixed and committed - please merge.
